src/usts/ewma.cpp

```diff
@@ -51,6 +51,16 @@ auto ewma<Clock>::warmed_up() const -> bool {
     return elapsed >= 35 * tau;
 }
 
+template<typename Clock>
+auto ewma<Clock>::calculate_alpha(time_point time) const -> double {
+    if (time < prev) {
```

Comment:
It seems that compile time assert with `Clock::is_steady` flag could be used somehow.